lib/data-helper.js

```diff
@@ -88,6 +88,39 @@ export function getAccountConfig( account ) {
 	return localConfig[ account ];
 }
 
+export function getAllAccountsWithFeatures( features = [] ) {
+	let allAccounts = [];
+
+	if ( config.has( 'accounts' ) ) {
+		allAccounts = config.get( 'accounts' );
+	} else if ( process.env.ACCOUNT_INFO ) {
+		allAccounts = JSON.parse( process.env.ACCOUNT_INFO );
+	}
+
+	return allAccounts.filter( function( account ) {
+		// return accounts that have all the features requested
+		return difference( account.features || [], features ).length === 0;
+	} );
+}
+
+export function hasAccountWithFeatures( features = [] ) {
+	if ( ! Array.isArray( features ) ) {
+		features = [ features ];
+	}
+
+	return getAllAccountsWithFeatures( features ).length > 0;
+}
+
+export function getRandomAccountWithFeatures( features = [] ) {
+	if ( ! Array.isArray( features ) ) {
+		features = [ features ];
+	}
+
+	const accounts = getAllAccountsWithFeatures( features );
+
+	return accounts[ Math.floor( Math.random() * ( accounts.length -1 ) ) ];
```

Comment:
Shouldn't -1 here, since [Math.random()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random) max is 0.9999999
> The Math.random() function returns a floating-point, pseudo-random number in the range 0 to 1 — that is, from 0 (inclusive) up to but not including 1 (exclusive)

so, for [1,2] you'll get `Math.floor( 0.999999 * 1 )`, never 1

Reply:
You're right